go/vt/vttest/local_cluster.go

```diff
@@ -381,6 +381,56 @@ func (hdl *Handle) MySQLConnParams() (mysql.ConnParams, error) {
 	return params, nil
 }
 
+// MySQLAppDebugConnParams builds the MySQL connection params for debug user.
+// It's valid only if you used MySQLOnly option.
+func (hdl *Handle) MySQLAppDebugConnParams() (mysql.ConnParams, error) {
+	params := mysql.ConnParams{
+		Charset: "utf8",
+		DbName:  hdl.dbname,
+	}
+	if hdl.Data == nil {
+		return params, errors.New("no data")
+	}
+	iuser, ok := hdl.Data["appdebug_username"]
```

Comment:
Looks like this is the only part different from `ConnParams`. It may be better to call that and then update params.Uname to `appdebug_username`.